### PR TITLE
Hamburger feature

### PR DESCRIPTION
**Pull request by Hugovs2000**
Finished up the Hamburger Menu making use of DaisyUI's drawer component. Also made the way I export functions more consistent.

### CODE REVIEW

**Review by theadley — COMMENTED** [review]
I'm not 100% sure about having <h2> elements inside your <ul>

File: src/modules/Footer/Footer.tsx
Comment (LesediMoko) [inline src/modules/Footer/Footer.tsx:1]: I'm probably missing this elsewhere but is there a reason why the footer's padding is a prop?

**Review by LesediMoko — APPROVED** [review]
Looks good, just have a question regarding one of the props

File: src/modules/Footer/Footer.tsx
Comment (Hugovs2000) [reply src/modules/Footer/Footer.tsx:1]: Ya so I am using the same footer in the hamburger menu as the one on the homepage so the padding needs to be different depending on where I use it. Thanks for asking though

File: src/modules/Drawer/components/ComicsList.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+}) {
+  return (
+    <ul className="mt-4 mb-2 text-emerald-600 text-lg">
+      {topData?.trending?.[7] && (
+        <DrawerTrendingSection trendingData={topData.trending?.[7]} />
+      )}
+      {latestUpdatesData && (
+        <DrawerLatestUpdatesSection latestUpdatesData={latestUpdatesData} />
+      )}
+      {seasonalData && <DrawerSeasonalSection seasonalData={seasonalData} />}
```
[the remark this answers is not in the document]
Comment (Hugovs2000) [reply src/modules/Drawer/components/ComicsList.tsx:24]: Yup, definitely. Blond moment

**Pull request merged**